README.md

```diff
@@ -6,6 +6,8 @@
 
 > 📦🔐 Pester is now signed. `-SkipPublisherCheck` should no longer be used to install from PowerShell Gallery on Windows 10.
 
+> 📦🔐 If you upgrade to 5.6.0 you will get a "certificate changed" error if you dont skip publisher check. This is related to a new module certificate. [more info here](#regarding-certificate-for-different-versions).
```

Comment:
```suggestion
> 📦🔐 Upgrading to 5.6.0 you show a "certificate changed" error, this is because a change in Root Certificate, and you have to specify `-SkipPublisherCheck` to update. [More info below.](#560-certificate-error).
```

Reply:
Replace "5.6.0 you" with "5.6.0 will" ?